templates/email_mentorship_accepted.html

```diff
@@ -0,0 +1,10 @@
+<p>Hi {{ user_name }},</p>
+<br>
+<p>Mentorship Relation accepted!</p>
+<br>
+<p>Congratulations, your Mentorship Relation has been accepted!</p>
```

Comment:
Can you please add who accepted the mentorship relationship? Also, having a link to the profile of the mentor would be good for quick actions!